packages/massa-web3/src/experimental/smartContract.ts

```diff
@@ -225,12 +233,92 @@ export class SmartContract {
     return new SmartContract(client, addr)
   }
 
-  // todo: implement call
+  /**
+   * Executes a smart contract call operation
+   * @param account - The account performing the operation.
+   * @param functionName - The smart contract function to be called.
+   * @param parameter - Parameters for the function call in Uint8Array or number[] format.
+   * @param options - Includes optional and required parameters like fee, maxGas, coins, and periodToLive.
+   * @returns A promise that resolves to an Operation object representing the transaction.
+   */
   async call(
-    _method: string, // eslint-disable-line @typescript-eslint/no-unused-vars
-    _args: Uint8Array, // eslint-disable-line @typescript-eslint/no-unused-vars
-    _opts: CallOptions // eslint-disable-line @typescript-eslint/no-unused-vars
-  ): Promise<Uint8Array> {
-    throw new Error('Not implemented')
+    account: Account,
+    functionName: string,
+    parameter: Uint8Array,
+    { fee, maxGas, coins = 0n, periodToLive }: CallOptions
+  ): Promise<Operation> {
+    await this.ensureBalance(account, coins)
+
+    fee = fee ?? (await this.client.getMinimalFee())
+
+    if (!maxGas) maxGas = await this.getGasEstimation()
```

Comment:
for code readability i would propose we always use brackets for code blocks..
if its ok for you we can activate the eslint rule for this:
https://archive.eslint.org/docs/2.0.0/rules/curly